Makefile

```diff
@@ -68,20 +68,30 @@ SPECREVISION = $(shell echo `git describe --tags --dirty`)
 SPECREMARK = from git branch: $(shell echo `git symbolic-ref --short HEAD`) \
 	     commit: $(shell echo `git log -1 --format="%H"`)
 endif
+# C++ for OpenCL doc revision scheme in aligned
+# with its release date.
+CXX4OPENCL_DOCREVISION = DOCREV$(shell echo `date +"%Y.%m"`)
```

Comment:
This would make the date the spec is built to be the identifier of a particular spec version.  I think this opens up too many opportunities for confusion: someone building an older revision of the repo will produce a document that "looks like" the most recent version while it is older.  I would strongly advise against using the build date alone as a document revision identifier.  Instead, I would suggest to use a variable that is manually updated with every release.

Reply:
Ok, thanks. This makes sense... other approaches could be to extract the revision from the tag... this is what we do for other specs... but however we add a `cxxforopencl` prefix to the tag of C++ for OpenCL document... maybe we don't want to add too much complexity in Makefile for extracting the subsrtings...

Reply:
I have updated this PR now to set the revision manually and updated the comments explaining the revision scheme.

For WIP doc builds we should just leave the revision blank.